chipy_org/apps/job_board/models.py

```diff
@@ -130,3 +122,47 @@ def days_elapsed(self):
             return days_elapsed_from_posting
```

Comment:
You can remove line 130 and just return delta.days.  Also would you want to return a timedelta object?  You seem to be casting the in to a timedelta most places.  
```python
return timedelta(days=delta.days)
```